src/table/components/__tests__/TableHeadWrapper.spec.tsx

```diff
@@ -124,33 +133,32 @@ describe('<TableHeadWrapper />', () => {
         { ...tableData.columns[0], sortDirection: 'desc' },
         { ...tableData.columns[1], sortDirection: 'asc' },
       ],
-      columnOrder: tableData.columnOrder,
-    };
+    } as unknown as TableData;
     layout = {
       qHyperCube: {
         qEffectiveInterColumnSortOrder: [1, 0],
       },
-    };
+    } as TableLayout;
 
-    const { queryByText } = renderTableHead();
-    const firstColQuery = queryByText(tableData.columns[0].label).closest('th');
-    const secondColQuery = queryByText(tableData.columns[1].label).closest('th');
+    const { getByText } = renderTableHead();
+    const firstColQuery = getByText(tableData.columns[0].label).closest('th');
+    const secondColQuery = getByText(tableData.columns[1].label).closest('th');
 
-    expect(firstColQuery.getAttribute('aria-sort')).toBeNull();
-    expect(firstColQuery.getAttribute('aria-pressed')).toBe('false');
-    expect(secondColQuery.getAttribute('aria-sort')).toBe('ascending');
-    expect(secondColQuery.getAttribute('aria-pressed')).toBe('true');
+    expect(firstColQuery?.getAttribute('aria-sort')).toBeNull();
+    expect(firstColQuery?.getAttribute('aria-pressed')).toBe('false');
+    expect(secondColQuery?.getAttribute('aria-sort')).toBe('ascending');
+    expect(secondColQuery?.getAttribute('aria-pressed')).toBe('true');
```

Comment:
Do need the ? here? I think getBy always returns an element, otherwise throws an error

Reply:
Can it? I think the test is checking if aria-sort is null. This should fail btw since aria-sort is undefined if it is not sorted on that column

Reply:
uh.. sorry forget my comment, doesn't matter using any of them... `firstColQuery` might be null anyway

Reply:
ah I get it, you are doing `closest` that can return null. I would say it is better to cast it, otherwise you don't know if it is the attribute or the element that is null. Do something like this:
`const firstColQuery = getByText(tableData.columns[0].label).closest('th') as HTMLTableCellElement;
    const secondColQuery = getByText(tableData.columns[1].label).closest('th') as HTMLTableCellElement;`

Reply:
Ok, I'll go for it

Reply:
now you should be able to remove the ?